draft-ysl-cats-metric-definition.md

```diff
@@ -94,67 +90,72 @@ This document uses the following terms defined in {{!I-D.ietf-cats-framework}}:
 
 # Definition of Metrics
 
-Definition and usage of specific metrics are related to the intended use case. However, when considering disseminating compute metrics to network devices, appropriate categorization and abstraction of CATS metrics is required in order to avoid introducing extra complexity into the network.
-
-This document defines three abstract metric levels to meet different requirements use cases listed in {{!I-D.ietf-cats-usecases-requirements}}:
+Introducing a definition of metrics necessitates balancing the following trade-off: if the metrics are too fine-grained, they become unscalable due to the excessive number of metrics that must be communicated through the protocol. Conversely, if the metrics are too coarse-grained, they may lack the necessary information to make informed decisions. To ensure scalability while providing sufficient detail for effective decision-making, we propose a definition of metrics that incorporates three levels of abstraction:
 
-- Level 0 (L0): Raw metrics. The metrics are not abstracted, so different metrics use their own unit and format as used within a compute orchestration domain.
+- **Level 0 (L0): Raw metrics.** These metrics are presented without abstraction, with each metric using its own unit and format as defined by the underlying resource.
 
-- Level 1 (L1): Normalized metrics in categories. The metrics are categorized into multiple dimensions, such as network, computing, and storage. Each category metric is normalized into a value or a set of values with a range of scores.
+- **Level 1 (L1): Normalized metrics in categories.** These metrics are derived by aggregating L0 metrics into multiple categories, such as network, computing, and storage. Each category is summarized with a single L1 metric by normalizing it into a value within a defined range of scores.
 
-- Level 2 (L2): Fully normalized metric. Metrics are normalized into a single value. The category information or raw metrics information cannot be interpreted from the value directly.
+- **Level 2 (L2): Fully normalized metric.** These metrics are derived by aggregating all the L1 metrics into a single L2 metric, which is then normalized into a value within a defined range of scores.
```

Comment:
I will argue the definition here. L2 can be generated from L0 directly as well? Logical and hierarchical layers of aggregation looks nice, but L1 is not required for L2. is it correct? An implementation can generate a L2 metric from L0 metrics without the L1 metrics

Reply:
I am ok with adding that flexibility and allowing L2 metrics to directly use L0 metrics. I will update the text in my next update.

Reply:
ok, thanks!